### PR TITLE
Feat(2882) : Chart collection V2

### LINKED ISSUE #2882: Better charts for collections v2
We would like definitely to see better charts for collections than we have currently.

Trying to conceptualize what you think would help people to make their decisions easier and faster, what kind of tip points you would like to see.
Feel free inspired by any others, we can probably cherry-pick any on-chain data thanks to SubSquid and make it fast

This would be an amazing task for someone who is a data scientist.

### Hint
- https://cryptoslam.io/cool-cats/sales/summary
-

### PR DESCRIPTION
**Pull request by MSghais**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [X] Bugfix
- [X] Feature
- [ ] Refactoring

### What's new?

- [X] Change first chart simple for lisibility.
 Add other Graph for specific stats.

### Before submitting Pull Request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality
- [X] I've posted a screenshot of demonstrated change in this PR

### Optional

- [X] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [X] I've tested PR on mobile and everything seems works
- [] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [X] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=5DkJ2x6SCHsXw9jarxhv3mVx1cuGsWBLK7pZNnvdRmguDuyy

### Community participation

- [X] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

### CODE REVIEW

**Comment by prachi00** [comment]
@vikiival @roiLeo can you please also take a look at this once?

**Review by roiLeo — COMMENTED** [review]
is it possible to put all data in the same graph? it's not very pretty to have two graphs one under the other

File: components/rmrk/Gallery/CollectionBarStats.vue [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    document!.getElementById('collectionPriceBar')!.style.cursor = 'auto'
+  }
+
+  protected onCanvasMouseLeaveBar(): void {
+    document!.getElementById('collectionPriceBar')!.style.cursor = 'auto'
+  }
+
+  public async created() {
+    window.addEventListener('resize', this.onWindowResize)
+  }
```
Comment (vikiival) [inline components/rmrk/Gallery/CollectionBarStats.vue:82]: I heard that a `ChartMixin` would a be good friend here

**Review by vikiival — CHANGES_REQUESTED** [review]
Some minor stuff

**Comment by MSghais** [comment]
> is it possible to put all data in the same graph? it's not very pretty to have two graphs one under the other

It is possible but not very readable. I tried a lot, but it's much more readable with simple graphs that are easily interpretable.

**Comment by prachi00** [comment]
@roiLeo @vikiival can we merge this one yet?

**Comment by vikiival** [comment]
~ So as I understand correctly the change is that @MSghais added the barchart?

**Comment by prachi00** [comment]
@MSghais any update on this one?

**Comment by prachi00** [comment]
> > ~ So as I understand correctly the change is that @MSghais added the barchart?
> 
> At the moment, I added the bar graph to represent more clearly two variables. The older graphic is pretty hard to read with 4 stats variables. That is why I divide that. I used all data available in rubick at this moment for the collection chart stats.
> 
> Let me know please

@vikiival

**Comment by vikiival** [comment]
> Let me know please

I was thinking about this issue for a couple of days and:
- your contribution is really nice
- like the bar chart for the value

but
- this won't close #2882 

finally:
- I've seen a couple of issues in deepsource, make sure they are resolved
- @prachi00 will do a code review and test that 🧪 
- once approved we can payout some bounty and happy to merge

**Comment by prachi00** [comment]
@MSghais can you resolve the conversations?

**Comment by MSghais** [comment]
Hi yes I try to solve the deepsource issues. 

But this rules is unecessary for the context of this PR and this commit, because document needed to checked strictly with non-null values, and also the function return void.  

Thanks for understanding @prachi00 @vikiival

**Review by prachi00 — APPROVED** [review]

**Comment by petersopko** [comment]
bump, @MSghais any update?

**Comment by yangwao** [comment]
Closing this as staled PR
Once @MSghais will 
- resolve deepsource issues 
- provide links where I can check and test it

happy to proceed.

**Pull request closed without merging**